src/payment/data/service.js

```diff
@@ -80,9 +79,10 @@ export function transformResults(data) {
   return results;
 }
 
-export async function getBasket() {
+export async function getBasket(discountJwt) {
+  const discountJwtArg = typeof discountJwt !== 'undefined' ? `?discount_jwt=${discountJwt}` : '';
   const { data } = await apiClient
-    .get(`${config.ECOMMERCE_BASE_URL}/bff/payment/v0/payment/`)
+    .get(`${config.ECOMMERCE_BASE_URL}/bff/payment/v0/payment/${discountJwtArg}`)
```

Comment:
the first purchase discount offer gets applied when a discount jwt is sent as an argument